book/internals/adders.nim

```diff
@@ -0,0 +1,36 @@
+import nimib, nimibook
+
+nbInit(theme = useNimibook)
+nbText: """
+## **Adders**
+
+If you remember the previous example with CustomLabel, not that you weren't able to write: 
```

Comment:
This sentence currently does not make sense to me. Maybe change it to something like `In the previous example, you were not able to write the following:`

Reply:
Ah, that should've been a "note that you weren't able to write", I'll reword it.